
bool circle_toggle
bool circle_in_progress
bool circle_clockwise
float64 circle_progress
float64 circle_angle
float64 circle_radius
int32 circle_state

bool swipe_toggle
bool swipe_in_progress
float64[3] swipe_direction
float64 swipe_speed
int32 swipe_state

bool keytap_toggle
bool keytap_in_progress
float64[3] keytap_direction
int32 keytap_state

bool screentap_toggle
bool screentap_in_progress
float64[3] screentap_direction
int32 screentap_state
